aidial_adapter_openai/utils/tokens.py

```diff
@@ -91,10 +98,18 @@ def discard_messages(
         len(kept_messages) == system_messages_count
         and system_messages_count != n
     ):
-        raise HTTPException(
-            message=f"The token size of system messages and the last user message ({prompt_tokens}) exceeds prompt token limit ({max_prompt_tokens})",
+        raise DialException(
+            message=(
+                f"The token size of system messages and the last user message ({prompt_tokens}) "
+                f"exceeds prompt token limit ({max_prompt_tokens})"
+            ),
             status_code=400,
             type="invalid_request_error",
+            display_message=(
+                f"The token size of system messages and the last user message ({prompt_tokens})"
```

Comment:
1. Let's introduce a variable to avoid repeating the same error message twice.
2. Please add a missing space before "Try".

Reply:
Done